CONTRIBUTING.md

```diff
@@ -1,31 +1,100 @@
-Contributing to EssentialsX
-===========================
+# Contributing to EssentialsX
 
-Want to help improve EssentialsX? There are several ways you can support and contribute to the project.
+*By contributing to EssentialsX, you agree to license your changes under the [GNU General Public License version 3](https://github.com/EssentialsX/Essentials/blob/2.x/LICENSE).*
 
-If you'd like to make a financial contribution to the project, you can join our [Patreon](https://www.patreon.com/essentialsx/),
-or to make a one-off donation you can visit our [Ko-fi page](https://ko-fi.com/essentialsx). If you can't make a
-donation, don't worry! There are lots of other ways to contribute:
+## Submitting code changes to EssentialsX
 
-* Do you run a server? Take a look at our ["help wanted"](https://github.com/EssentialsX/Essentials/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22help+wanted%22)
-  and ["bug: unconfirmed"](https://github.com/EssentialsX/Essentials/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22bug%3A+unconfirmed%22)
-  issues, where you can find issues that need extra testing and investigation.
-* Do you speak multiple languages? If so, we always welcome contributions to our [Crowdin project](https://crowdin.com/project/essentialsx-official).
-* Do you enjoy helping others? If so, why not contribute to the [EssentialsX documentation](https://github.com/EssentialsX/wiki)?
-  You can also join the [MOSS Discord community](https://discord.gg/casfFyh) and provide direct community support to
-  other EssentialsX users.
-* If you're a developer, you could look through our ["open to PR"](https://github.com/EssentialsX/Essentials/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22status%3A+open+to+PR%22)
-  issues. We're always happy to receive bug fixes and feature additions as pull requests.
+If you're interested in submitting new code to EssentialsX, we accept changes via GitHub pull requests.
 
-Submitting a PR
----------------
+### Adding features
 
-EssentialsX has extensive templates for PRs that detail how to submit your PR. To find out more, see the PR templates for:
-* [Bug fixes](https://github.com/EssentialsX/Essentials/blob/2.x/.github/PULL_REQUEST_TEMPLATE/bug-fix.md) 
-* [New features](https://github.com/EssentialsX/Essentials/blob/2.x/.github/PULL_REQUEST_TEMPLATE/new-feature.md)
+In general, we will only consider PRs for features if they have already been discussed with the team through
+GitHub issues and discussions. Check the list of
+["open to PR" issues](https://github.com/EssentialsX/Essentials/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22status%3A+open+to+PR%22)
+and [feature request discussions](https://github.com/EssentialsX/Essentials/discussions/categories/ideas-and-feature-suggestions)
```

Comment:
> This might lead people to open duplicates of already-rejected feature requests though?

We're telling them to only open PRs for open feature requests

Reply:
The next sentence says to open a feature request if they don't see one

Reply:
:trollface: